src/lib/Scenes/Artwork/Components/RequestConditionReport.tsx

```diff
@@ -0,0 +1,186 @@
+import { Button, Flex } from "@artsy/palette"
+import { RequestConditionReport_artwork } from "__generated__/RequestConditionReport_artwork.graphql"
+import { RequestConditionReport_me } from "__generated__/RequestConditionReport_me.graphql"
+import { RequestConditionReportMutation } from "__generated__/RequestConditionReportMutation.graphql"
+import { RequestConditionReportQuery } from "__generated__/RequestConditionReportQuery.graphql"
+import { Modal } from "lib/Components/Modal"
+import { defaultEnvironment } from "lib/relay/createEnvironment"
+import { Schema, track } from "lib/utils/track"
+import React from "react"
+import { View } from "react-native"
+import { commitMutation, createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import { PayloadError } from "relay-runtime"
+
+interface RequestConditionReportProps {
+  artwork: RequestConditionReport_artwork
+  me: RequestConditionReport_me
+}
+
+interface State {
+  requestingConditionReport: boolean
+  showConditionReportRequestedModal: boolean
+  showErrorModal: boolean
+  errorModalText: string
+}
+
+@track({
+  action_name: Schema.ActionNames.RequestConditionReport,
+  context_module: Schema.ContextModules.ArtworkDetails,
+})
+export class RequestConditionReport extends React.Component<RequestConditionReportProps, State> {
+  state = {
+    requestingConditionReport: false,
+    showConditionReportRequestedModal: false,
+    showErrorModal: false,
+    errorModalText: "",
+  }
+
+  requestConditionReport = () => {
+    const { artwork } = this.props
+    return new Promise(async (resolve, reject) => {
+      commitMutation<RequestConditionReportMutation>(defaultEnvironment, {
+        onCompleted: data => {
+          resolve(data)
+        },
+        onError: error => {
+          reject(error)
+        },
+        mutation: graphql`
+          mutation RequestConditionReportMutation($input: RequestConditionReportInput!) {
+            requestConditionReport(input: $input) {
+              conditionReportRequest {
+                internalID
+              }
+            }
+          }
+        `,
+        variables: {
+          input: { saleArtworkID: artwork.saleArtwork.internalID },
+        },
+      })
+    })
+  }
+
+  @track({
+    action_type: Schema.ActionTypes.Fail,
+  })
+  presentErrorModal(errors: Error | ReadonlyArray<PayloadError>, mutationMessage: string) {
```

Comment:
looks likie this `mutationMessage` value is always `null`, might as well delete it?

Reply:
Yep, good catch